Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM rust:1.54 as builder
+FROM rust:slim-bullseye as builder
 RUN curl -L https://yt-dl.org/downloads/2021.06.06/youtube-dl -o /usr/local/bin/youtube-dl && chmod a+rx /usr/local/bin/youtube-dl
 WORKDIR /usr/src/mrvn-bot
+RUN apt-get update && apt-get install -y libopus0 libopus-dev pkg-config ffmpeg && rm -rf /var/lib/apt/lists/*
 COPY . .
 RUN cargo install --path ./mrvn-front-discord
 
-FROM debian:buster-slim
-RUN apt-get update && apt-get install -y ca-certificates ffmpeg python && rm -rf /var/lib/apt/lists/*
+FROM bitnami/minideb:latest
```

Comment:
This should probably use a stable tag, ie `bitnami/minideb:buster`, although looking at this now we should probably also be using debian stretch which is LTS.

Reply:
Alright, it will use `bitnami/minideb:buster` then

Reply:
>I'm not totally convinced there's a need to use a slim image here, especially since it means we need to manually install a bunch of packages. Is there a good reason to change it?

On builder side it is just needs according to songbird docs only ffmpeg and libopus dependencies including -dev one for libopus.